index.html

```diff
@@ -146,11 +146,16 @@ <h2 class="title">Bicep Workout</h2>
         </div>
       </div>
       <div class="card">
+        <a href="./cardio.html">
         <div class="img">
           <img src="./images/cardio_day.jpg" alt="" />
         </div>
+      </a>
         <div class="content">
-          <h2 class="title">Cardio Workout</h2>
+          <a href="./cardio.html">
+            <h2 class="title">Cardio Workout</h2>
+          </div>

```

Comment:
Why you are closing div in this section

Reply:
Remove it and push it again

Reply:
Now, it has been resolved. @TusharKesarwani

Reply:
It was a typing error.